README.md

```diff
@@ -286,6 +166,5 @@ I would like to thank...
 
 This project is [MIT](./LICENSE) licensed.
 
-_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
Please note that the MIT Lisence link provided is not valid 😢.  Kindly add a valid link 👍🏼
![Screenshot_20230329_124630](https://user-images.githubusercontent.com/44624138/228526795-805c3461-ad53-484e-aaba-c5f58ffb57ee.jpg)

Reply:
Let me fix that